sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/FieldSingleDeclarationCheck_OK.java

```diff
@@ -0,0 +1,10 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FieldSingleDeclarationCheck_OK {
+
+    org.slf4j.Logger logger;
+
+}
```

Comment:
I do not see tests for cases with inner classes (in class , in method, ..... yes real life code is crazy on purpose).
If we care only about Top level class we should define this in javadoc or .... .